Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnect.cs

```diff
@@ -55,96 +47,49 @@ private set
 			}
 		}
 
-		public bool ConnectOnStart
-		{
-			set => _connectOnStart = value;
-		}
-
-		public bool ConnectOnAwake
-		{
-			set => _connectOnAwake = value;
-		}
-
-		public ClientMeta AppData
-		{
-			set => _appData = value;
-		}
-
 		public WalletConnectEventWithSessionData ConnectedEvent => _connectedEventSession;
 
-	#if UNITY_EDITOR
-		public NativeWebSocketTransport Transport
+		public WalletConnect()
 		{
-			set => _transport = value;
 		}
-	#endif
-		private static WalletConnect Instance { get; set; }
 
-		private async void Awake()
+		public void Initialize(string settingsFileName = "WalletConnectSettings")
```

Comment:
Let's not bother wallet connect with loading its settings. Initialize should already receive acquired 
`WalletConnectSettingsSO `